subsys/bluetooth/host/gatt.c

```diff
@@ -4171,6 +4189,9 @@ int bt_gatt_clear(u8_t id, const bt_addr_le_t *addr)
 		return err;
 	}
 
+#if defined(CONFIG_BT_GATT_CLIENT)
```

Comment:
Make this check interval to bt_gatt_clear_subscriptions so we don't have it in multiple places, the compiler will optimize it out if there is no code inside.

Reply:
I pushed a new version now, is that how you meant?